spec/support/unit/helpers/active_record_versions.rb

```diff
@@ -50,5 +50,9 @@ def active_record_supports_expression_indexes?
     def active_record_supports_validate_presence_on_active_storage?
       active_record_version >= '6.0.0.beta1'
     end
+
+    def active_record_supports_has_rich_text?
+      active_record_version >= '6.0.0.beta1'
```

Comment:
It's possible to have Rails 6 and not load Action Text:

https://github.com/rails/rails/blob/34d66ca6bab109540299a7b30980627bea37e323/railties/lib/rails/all.rb#L16

So to check if Action Text is available, you can check for `ActionText::RichText` or `ActionText::Engine`.

Reply:
You're right!

I'll add this validation!

Reply:
I updated to this:
```
    def active_record_supports_has_rich_text?
      active_record_version >= '6.0.0.beta1' &&
        defined?(ActionText::RichText)
    end
```
Was that what you had in mind?

Reply:
I don't think you need to check the rails version (some people are even backporting ActionText to 5.2).
Just `defined?(ActionText::RichText)`should be enough 👍